application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/ECChronosInternals.java

```diff
@@ -109,6 +109,7 @@ public ECChronosInternals(final Config configuration,
                     .withMetricRegistry(metricRegistry)
                     .withJmxReporting(configuration.getStatistics().getReporting().isJmxReportingEnabled())
                     .withFileReporting(configuration.getStatistics().getReporting().isFileReportingEnabled())
```

Comment:
These might also be able to be refactored so just statistics needs to be passed

Reply:
Statistics is in application module, application depends on core, connection etc. We can't use statistics in core because that will lead to circular dependency.

Reply:
It would still be good to pair metrics into it's own object since we are sending it into a Connection but I am fine without it.